ocs_ci/ocs/resources/pod.py

```diff
@@ -300,6 +300,55 @@ def run_git_clone(self):
         assert wl.setup(), "Setup up for git failed"
         wl.run()
 
+    def install_packages(self, packages):
+        """
+        Install packages in a Pod
+
+        Args:
+            packages (list): List of packages to install
+
+        """
+        if isinstance(packages, list):
+            packages = ' '.join(packages)
+
+        cmd = f"yum install {packages} -y"
+        self.exec_cmd_on_pod(cmd, False)
+
+    def copy_to_node(self, server, authkey, localpath, remotepath, user=None):
+        """
+        Upload a file from pod to server
+
+        Args:
+            server (str): Name of the server to upload
+            authkey (str): Authentication file (.pem file)
+            localpath (str): Local file in pod to upload
+            remotepath (str): Target path on the remote server
+            user (str): User name to connect to server
+
+        """
+        if not user:
+            user = "root"
+
+        cmd = f"scp -i {authkey} {localpath} {user}@{server}:{remotepath}"
```

Comment:
If we will add -r option it will allow us to copy also directories.

Reply:
If I use -r option , I am getting Identity issue . 
```
sh-4.2# scp -i /tmp/libra.pem /f1 test@rhel1-0.nov061.qe.rh-ocs.com:/tmp/
f1                                                                                                                                                          100%    0     0.0KB/s   00:00    
sh-4.2# 
sh-4.2# 
sh-4.2# scp -ir /tmp/libra.pem /f1 test@rhel1-0.nov061.qe.rh-ocs.com:/tmp/
Warning: Identity file r not accessible: No such file or directory.
Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
lost connection
sh-4.2# scp -ir /tmp/libra.pem /test test@rhel1-0.nov061.qe.rh-ocs.com:/tmp/
Warning: Identity file r not accessible: No such file or directory.
Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
lost connection
sh-4.2# 
```

I will debug later and take it in next PR

Reply:
after -i you need to have key so that's the issue here.  So you need to make it like
`scp -i /tmp/libra.pem -r ...`

Reply:
Done